tools/notification-configuration/common/Services/AzureDevOpsService.cs

```diff
@@ -179,7 +179,9 @@ public async Task<string> GetDescriptorForPrincipal(string userPrincipal)
         {
             var client = await GetClientAsync<GraphHttpClient>();
 
-            logger.LogInformation("GetDescriptorForAlias UserPrincipal = {0}", userPrincipal);
+            // Do not log PII: "alias@microsoft.com" => "***@microsoft.com"
+            var loggableUserPrincipal = $"***{userPrincipal.Substring(userPrincipal.IndexOf("@"))}";
```

Comment:
This changes seems unrelated.

Reply:
Was a strange rebase artifact. Reverted. It'll go away in the squash.